packages/modals/src/styled/StyledClose.ts

```diff
@@ -5,89 +5,26 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { css, DefaultTheme, ThemeProps } from 'styled-components';
-import {
-  getColorV8,
-  retrieveComponentStyles,
-  DEFAULT_THEME,
-  focusStyles
-} from '@zendeskgarden/react-theming';
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { IconButton } from '@zendeskgarden/react-buttons';
 
 const COMPONENT_ID = 'modals.close';
 
-const colorStyles = (props: ThemeProps<DefaultTheme>) => {
-  const backgroundColor = 'primaryHue';
-  const foregroundColor = 'neutralHue';
-
-  return css`
-    background-color: transparent;
-    color: ${getColorV8(foregroundColor, 600, props.theme)};
-
-    &:hover {
-      background-color: ${getColorV8(backgroundColor, 600, props.theme, 0.08)};
-      color: ${getColorV8(foregroundColor, 700, props.theme)};
-    }
-
-    ${focusStyles({
-      theme: props.theme,
-      color: { hue: backgroundColor }
-    })}
-
-    &:active {
-      /* prettier-ignore */
-      transition:
-        background-color 0.1s ease-in-out,
-        color 0.1s ease-in-out;
-      background-color: ${getColorV8(backgroundColor, 600, props.theme, 0.2)};
-      color: ${getColorV8(foregroundColor, 800, props.theme)};
-    }
-  `;
-};
-
 export const BASE_MULTIPLIERS = {
   top: 2.5,
   side: 6.5,
   size: 10
 };
 
-/**
- * 1. Remove dotted outline from Firefox on focus.
- */
-export const StyledClose = styled.button.attrs({
+export const StyledClose = styled(IconButton).attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
 })`
```

Comment:
Keeping as is for now even though attributes set on `StyledIconButton` take precedence.

https://github.com/zendeskgarden/react-components/blob/52822988c09ec0bbcf0a82f63d3d8622b74d0b98/packages/buttons/src/styled/StyledIconButton.ts#L72-L74

Reply:
Hey @ze-flo, I have a [commit](https://github.com/zendeskgarden/react-components/pull/1807/commits/0171530849f0fe2408491af36caaf8de39667150) on the buttons PR that resolves the issue 🙌